bin/questions.js

```diff
@@ -21,6 +19,92 @@ const QUESTIONS = [
             'componentDidCatch'
         ]
     }
-]
+];
+
+const correctDir = (confirm = 'correctDir') => {
+    return (answers) => {
+        return answers[confirm];
+    }
+};
+
+const validateData = () => {
+    return (value) => {
+        return (value && value !== '') ? true : 'That is not a valid entry, please try again.';
+    }
+};
+
+const checkForProject = () => {
+    return (answers) => {
+        return (!answers.correctDir || projectName) ? false : true;
+    }
+};
+
+const changeProject = () => {
+    return (answers) => {
+        return (answers.correctDir && !answers.projectName) ? true : false;
+    }
+}
+
+const setupQuestions = [
+    {
+        type: 'confirm',
+        message: 'Would you like to auto-generate tests?',
+        name: 'autoGenerateTests'
+    },
```

Comment:
I would move this further down the path.... I think the first question we want to ask is if the cwd is the correct directory and then set up a project name.... probably move this close to the  bottom, below the component dir question?

Reply:
It flows better, you're right.